### ISSUE TITLE
[Bug]: Mahou Tsukai - Caliburn & Morgan not working

### ISSUE THREAD

**Issue by EtrnaL5** [AllTheMods/ATM-10#1605]
### Possible Fixes

Yes

### Modpack Version

All The Mods 10-2.13

### What happened?

Hey, I was trying to upgrade the Caliburn with ender orb upgrades and it was not working as intended, I then tried nether stars and it worked perfectly fine but only increased the innate cap to 60 I then tried to upgrade to Morgan by killing the Warden and it never upgraded.

**Comment by HelixRanmaru** [comment]
@EtrnaL5 in ATM 10, its back to the default of Netherstars to increase the innate cap and to kill a tamed dog to convert it to Morgan. The quest is currently incorrect and should be corrected in the next update.

**Comment by FreddyGoodman** [comment]
So:
- 60 damage cap (+ Apotheosis, Enchanting, etc.)
- Netherstars to increase the cap
- Killing a tamed dog converts it to Morgan.

And that is also how it's meant to be and this won't change in the future? I'm not asking for it to be changed. Just want to make sure it makes sense crafting it now. If I want to, I can just change the config.

**Comment by HelixRanmaru** [comment]
@FreddyGoodman Yes, as of this moment and time, this is how it's meant to be in ATM 10.

**Comment by EtrnaL5** [comment]
Thank you so much for the clarification! Hope you have a good day :)

**Comment by weeaudi** [comment]
i noted that the quest (upgrading caliburn) says 60 total nether stars but i believe that to be wrong 
```
POWER_CONSOLIDATION_TIER_UPGRADE_ITEM = ["minecraft:nether_star"]
#This is the range for undead mobs to fear Caliburn
# Default: 15
# Range: 0 ~ 100000000
POWER_CONSOLIDATION_FEAR_RADIUS = 15
#Power Consolidation Damage Cap Tiers
POWER_CONSOLIDATION_TIERS = [30.0, 35.0, 40.0, 45.0, 50.0, 55.0, 60.0]
````

there are only 7 tiers needing only 6 nether stars

**Comment by CaptainChaos21** [comment]
hi im using atm10 and my dog is invincible and wont take any damage

**Comment by double-pi** [comment]
@CaptainChaos21 Still happens to you? Cuz for me it's fine

**Comment by double-pi** [comment]
Also, for me killing villagers doesn't increase the Morgan's damage? Does it happen to anyone else?
